internal/pkg/v2/bootstrap/container/container.go

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/core/data/config"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/controller/errorconcept"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/interfaces"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+
+	"github.com/edgexfoundry/go-mod-messaging/messaging"
+)
+
+type Container struct {
+	ConfigClient       *config.ConfigurationStruct
```

Comment:
Same issue with using core data's specific ConfigurationStruct.

Reply:
Move to `/internal/core/data/v2 `.